src/extension.ts

```diff
@@ -191,6 +193,8 @@ export function activate(context) {
     subscriptions.forEach((element) => {
         context.subscriptions.push(element);
     }, this);
+
+    return registerYamlSchemaSupport();
```

Comment:
`registerYamlSchemaSupport` is async so returns a promise - I don't think we want to return that from `activate` do we?

Reply:
registerYamlSchemaSupport  is async, the result is not import and should be discarded.